packages/app-store/facetime/config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "/*": "Don't modify slug - If required, do it using cli edit command",
+  "name": "Facetime",
+  "title": "Facetime",
+  "slug": "facetime",
+  "type": "facetime_video",
+  "imageSrc": "/api/app-store/facetime/icon.svg",
+  "logo": "/api/app-store/facetime/icon.svg",
+  "url": "https://cal.com/apps/facetime",
+  "variant": "conferencing",
+  "categories": ["video"],
+  "publisher": "Mythie",
+  "email": "help@cal.com",
+  "description": "Facetime makes it super simple for collaborating teams to jump on a video call.",
+  "__createdUsingCli": true,
+  "appData": {
+    "location": {
+      "linkType": "static",
+      "type": "integrations:facetime_video",
+      "label": "Facetime",
+      "organizerInputPlaceholder": "https://facetime.apple.com/join... or facetime:<contact>",
```

Comment:
facetime:<contact> would only work if the facetime app is installed. Otherwise this link would just not work.

So, I think we shouldn't suggest that. If facetime works through browser that's the right thing to do. 
I hope facetime website would automatically open facetime app if it's installed.

Reply:
It does open the app when visiting the website, however that still requires that a user generate a FaceTime link. If we’re happy for that to be required I can make the update in a few hours once it’s morning again.

Reply:
I think that would be a better approach because the organizer can generate the link once. If we allow **facetime:** scheme, bookers won't be able to join meeting by clicking on the link they receive.

Reply:
Latest commit updates the config to follow the new decision on the FaceTime links, the placeholder is also updated correspondingly:

<img width="585" alt="image" src="https://user-images.githubusercontent.com/13398220/219810417-830a1229-0786-4d83-940f-a19ab33684cd.png">